rotkehlchen/chain/evm/decoding/constants.py

```diff
@@ -19,6 +19,7 @@
 ERC20_OR_ERC721_TRANSFER = b'\xdd\xf2R\xad\x1b\xe2\xc8\x9bi\xc2\xb0h\xfc7\x8d\xaa\x95+\xa7\xf1c\xc4\xa1\x16(\xf5ZM\xf5#\xb3\xef'  # noqa: E501
 # keccak of approve(address,uint256)
 ERC20_APPROVE = b'\x8c[\xe1\xe5\xeb\xec}[\xd1OqB}\x1e\x84\xf3\xdd\x03\x14\xc0\xf7\xb2)\x1e[ \n\xc8\xc7\xc3\xb9%'  # noqa: E501
+ERC1155_TRANSFER_SINGLE = b'\xc3\xd5\x81h\xc5\xaes\x97s\x1d\x06=[\xbf=exTBsC\xf4\xc0\x83$\x0fz\xac\xaa-\x0fb'  # noqa: E501
```

Comment:
Why not add this to `_maybe_decode_erc20_721_transfer`? Maybe is good idea to discuss this with lefteris too since is bigger change in discord but as you have now seems more like a patch in the ens decoder where you gnerate a new event right? that is what `decode_erc20_721_transfer` does afaik

Reply:
Discussed in discord. This will be handled separately